library/entropy_poll.c

```diff
@@ -54,28 +54,41 @@
 #define _WIN32_WINNT 0x0400
 #endif
 #include <windows.h>
-#include <wincrypt.h>
+#include <bcrypt.h>
+#if defined(_MSC_VER)
```

Comment:
Shouldn't it be `#if _MSC_VER <= 1600`?
I believe visual studio 2010 `_MSC_VER` value is 1600

Reply:
Yes, that's a good idea. That way we'll catch any legitimate instances of duplicated definitions with later versions of the compiler. I've overwritten the last commit with this change.